python/lsst/faro/base/CatalogMeasurementBase.py

```diff
@@ -126,6 +128,8 @@ def _getReferenceCatalog(self, butlerQC, dataIds, refCats, filterList, epoch=Non
         refCatCorrected : `numpy.ndarray`
             Catalog of reference objects with proper motions and color terms
             (optionally) applied.
+        refCatFrame: pandas.dataframe
```

Comment:
The returns have changed such that `refCat` and `refCatCorrected` are no longer being returned. In that case, these should be removed from the list of returns.

Reply:
Agreed this docstring has been changed with a longer description at the beginning and the following returns 
>Returns
>        -------
>        refCat: pandas.dataframe
>            a reference catalog with original columns and corrected
>            coordinates (ra,dec) and reference magnitudes  (refMag-/refMagErr)